homeassistant/components/frontend/__init__.py

```diff
@@ -24,6 +24,8 @@
 CONF_THEMES = 'themes'
 CONF_EXTRA_HTML_URL = 'extra_html_url'
 CONF_EXTRA_HTML_URL_ES5 = 'extra_html_url_es5'
+CONF_EXTRA_MODULE_URL = 'extra_module_url'
+CONF_EXTRA_MODULE_URL_ES5 = 'extra_module_url_es5'
```

Comment:
This is not 'module', because we import it as a normal JS script. So we should call it `extra_js_url_es5` (just to emphasize it's only for older ES5 builds)